src/OrbitCaptureClient/CaptureEventProcessor.cpp

```diff
@@ -79,6 +79,9 @@ void CaptureEventProcessor::ProcessEvent(const ClientCaptureEvent& event) {
     case ClientCaptureEvent::kModuleUpdateEvent:
       // TODO (http://b/168797897): Process module update events
       break;
+    case ClientCaptureEvent::kTotalMemoryUsageInfo:
+      // TODO: Process the total memory usage information.
```

Comment:
Please add a bug id (or at least your LDAP).

Reply:
Thanks for the suggestion. I added the corresponding bug here. Please refer to the commit *[[fixup] Adjust the field numbers in ClientCaptureEvent](https://github.com/google/orbit/pull/1918/commits/9375c9730a4c3547ce0778f932d5df17178ee99e)*.